part-2-server/server.js

```diff
@@ -0,0 +1,34 @@
+var http = require('http')
+var fs = require('fs')
+
+var server = http.createServer(function(request, response){
+  var url = request.url;
+
+  if(url === "/"){
+      fs.readFile('index.html', function(err, data){
+        if(err) throw err;
+        // else{
+        response.write(data.toString())
+            response.end()
+      // }
+      })    
+  }
+
+  else if(url === "/page-2"){
+      fs.readFile('page-2.html', function(err, data){
+        response.write(data.toString())
+        response.end()
+      })
+  }
+  else if(url === "/page-3"){
+      fs.readFile('page-2.html', function(err, data){
+        response.write(data.toString())
+        response.end()
+
+      })
```

Comment:
Need tp break the poem url into two separate substrings. The first part of the substring should be 'poem-', the second part should be the number. Then if the request-url starts with 'poem-' you respond with the correct file based on the number